MinimalCirclesPath.py

```diff
@@ -67,20 +67,26 @@ def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
 
     # TODO For actual application you would need to capture N/S/E/W
 
-    # list of coordinates returned to call API with
-    coordList = [(startLat, startLong)]
+    
       
     # gets total distance from start to finish
     totalDistance = getDistance(startLat, startLong, destLat, destLong)
 
     # gets step distance (pythagorean theorum)
-    stepDistance = sqrt((radius)**2-(pathWidth/2)**2)
+    stepDistance = 2 * sqrt((radius)**2-(pathWidth/2)**2)
```

Comment:
obviously some kind of bug or problem was found which required this change ... are there unit tests for this function?  do we have an example start/dest for which the intermediate points this function calculates are known a priori and verified?

this is exactly the kind of complex code (coding a complex algorithm) for which unit tests are ideally suited, the implementation of this algorithm has to be "right" or the usefulness of this whole program becomes effectively zero

Reply:
I plan on making unit test that's next on my todo list